src/toil/jobStores/aws/jobStore.py

```diff
@@ -681,6 +688,21 @@ def bucket_creation_pending(e):
                             raise
                         else:
                             return None
+                    elif e.status == 301:
+                        # This is raised if the user attempts to get a bucket in a region outside
+                        # the specified one, if the specified one is not `us-east-1`.  The us-east-1
+                        # server allows a user to use buckets from any region.
+                        if self.region == 'us-east-1':
+                            message = ('Region is us-east-1 but 301 error was raised when '
```

Comment:
I think that we don't need to make the message that detailed. The message to the user should be that a bucket of the given name exists but is not the the specified reason. The us-east-1 thing is a mere implementation detail.